src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+const icon1 = "images/icon1.png";
+const icon2 = "images/icon2.png";
+const icon3 = "images/icon3.png";
+const icon4 = "images/icon4.png";
+import "./Footer.css";
+
+const Footer = () => {
+  return (
+    <div className="footer">
+      <div className="card">
+        <img src={icon1} />
+        <h4>Declarative</h4>
+        <p>React makes it painless to create interactive UIs.</p>
+      </div>
```

Comment:
As a bonus, you could try and refactor these divs into a component "Card" and call it here, passing the information on it as props. 👍